asp_interactive_configuration/poetry.lock

```diff
@@ -294,89 +294,183 @@ files = [
+    {file = "clingcon-5.2.1.post4-cp39-cp39-musllinux_1_1_x86_64.whl", hash = "sha256:051024323963e9dfcade726bd7dea402c75a778f50733efd600efc712ae94d37"},
+    {file = "clingcon-5.2.1.post4-cp39-cp39-win32.whl", hash = "sha256:13d2141871a95d39323a552c62870168ffd2049b22d20be582c6bb03fcbbad09"},
+    {file = "clingcon-5.2.1.post4-cp39-cp39-win_amd64.whl", hash = "sha256:2caeccf31a2d48b97bee40b162bdd20aea54f85d3db679809d821086d18dc92d"},
+    {file = "clingcon-5.2.1.post4-pp37-pypy37_pp73-macosx_10_9_x86_64.whl", hash = "sha256:cd97522f74d6e5e0ca691c3544e676e208cad1b40a822979f06168c800e466f3"},
+    {file = "clingcon-5.2.1.post4-pp37-pypy37_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:54011b176435a84725b057dc3e20a92d2428ebfc9d52d2096c51b52f416bd5b6"},
+    {file = "clingcon-5.2.1.post4-pp37-pypy37_pp73-manylinux_2_17_i686.manylinux2014_i686.whl", hash = "sha256:32c5e34b547ca79a45cc32ea3d8e4b8464bb86073135849ba5b3725d4885aa89"},
+    {file = "clingcon-5.2.1.post4-pp37-pypy37_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:be39a59e915bbf218dfa05b3fc176feba27ff30e3ce6701769f739953a0d0705"},
+    {file = "clingcon-5.2.1.post4-pp38-pypy38_pp73-macosx_10_9_x86_64.whl", hash = "sha256:d70c93d6838f67ae4f8741e975ca938de8a53f1c1c5d663d202039f4552c32dc"},
+    {file = "clingcon-5.2.1.post4-pp38-pypy38_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:ba1cc630f8ad4b17cb505199ea0045e66cc257e97c55357d52b8653257d40b38"},
+    {file = "clingcon-5.2.1.post4-pp38-pypy38_pp73-manylinux_2_17_i686.manylinux2014_i686.whl", hash = "sha256:3438796bc226eb68dacff097182e9ab28c05adcbaeb3d6c4f2f16dd78fc86913"},
+    {file = "clingcon-5.2.1.post4-pp38-pypy38_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:49f478ae4b753b3fc6d44d84148067e8995ca1f56d7b719311b50f9b89a12570"},
+    {file = "clingcon-5.2.1.post4-pp39-pypy39_pp73-macosx_10_9_x86_64.whl", hash = "sha256:5be2961e12d4115af8c0506e39184c65aab2eb2c197d857cd7e75493d900ed7c"},
+    {file = "clingcon-5.2.1.post4-pp39-pypy39_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:e7ef22206d5fe763744d7282c59c0fdb0fa35b4b726d0fc967c0daa8694bf2f3"},
+    {file = "clingcon-5.2.1.post4-pp39-pypy39_pp73-manylinux_2_17_i686.manylinux2014_i686.whl", hash = "sha256:fd8334409d88c2283e34e58fc14f453c80faf4c22fc1f42d6e5a19840b8931a6"},
+    {file = "clingcon-5.2.1.post4-pp39-pypy39_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:a0a66c656a6d2e63d4317b4be9b9b275e2e4d8ad3b4ee95b98f0a056aef9ad51"},
+    {file = "clingcon-5.2.1.post4.tar.gz", hash = "sha256:8b8fbc659fb266b1d0dc7cb79a7c009e2fa321a72acb86c94f1ebda8dc824866"},
+]
+
+[package.dependencies]
+cffi = "*"
+clingo = "*"
+
+[package.source]
+type = "legacy"
+url = "https://test.pypi.org/simple"
+reference = "test"
+
 [[package]]
 name = "clingo"
-version = "5.6.2"
+version = "5.7.0.post9"
```

Comment:
`poetry install` gives me:

> RuntimeError
> Unable to find installation candidates for clingo (5.7.0.post9)

Reply:
This is because we need clingo 5.7 for fclingo to work. This version is only under pypi.test. I am not sure how to add this additional url to the poetry file. I will take a look.